pom.xml

```diff
@@ -1804,6 +1804,11 @@
                 <artifactId>json-smart</artifactId>
                 <version>1.3</version>
             </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon</groupId>
+                <artifactId>org.wso2.carbon.identity.totp</artifactId>
+                <version>${project.version}</version>
```

Comment:
Do not use project.version instead use the property defined. ex- carbon.identity.version